src/bin/pg_dump/pg_dump.c

```diff
@@ -2149,6 +2149,14 @@ dumpTableData_insert(Archive *fout, const void *dcontext)
 			continue;
 		if (tbinfo->attgenerated[i] && dopt->column_inserts)
 			continue;
+
+		/* rowversion/timestamp columns need to be skipped as instead of dumping they should be re-generated */
```

Comment:
Let's rephrase the message like:
`Skip TSQL ROWVERSION/TIMESTAMP column, it should be re-generated during restore.`